force-app/main/default/classes/ESA_cls_ExternalCommitteeTriggerHandler.cls

```diff
@@ -0,0 +1,61 @@
+/**********************************************************************************************
+* @Author:      Vasanthi D 
+* @Date:        11 Sep 2023
+* @Description: The purpose of this class is to create methods which can be used by different trigger event
+* @Revision(s): [Date] - [Change Reference] - [Changed By] - [Description]   
+***********************************************************************************************/
+public with sharing class ESA_cls_ExternalCommitteeTriggerHandler {
+    public static boolean cloneOpenCases_runsOne = TRUE;
+    public static void cloneOpenCases(List<External_Committee__c> newMeetings){
+        cloneOpenCases_runsOne = FALSE;
+        set<id> parentIds  = new set<id>();
```

Comment:
@deepakmulamalla : In the next line, map is initialized as `Map` but in this line set is not. Can we be consistent?

Reply:
Also for the map values that will have list.